Update to use page.emulateNetworkConditions (#17)
addyosmani/puppeteer-webperf: README.md

<h3 id="throttle-network">Emulate a slow network</h3>

> If you need to throttle the network connection, use `page.emulateNetworkConditions`. You can find the available parameters
> in the <a href="https://github.com/puppeteer/puppeteer/blob/v8.0.0/docs/api.md#pageemulatenetworkconditionsnetworkconditions">puppeteer documentation</a>.

Reviewer: ```suggestion
If you need to throttle the network connection, use [Puppeteer’s `page.emulateNetworkConditions` API](https://github.com/puppeteer/puppeteer/blob/v8.0.0/docs/api.md#pageemulatenetworkconditionsnetworkconditions).
```

<h3 id="throttle-network-cpu">Emulate a slow network and CPU</h3>

> CPU throttling allows you to simulate how a page performs on slower mobile devices. This can be done using `page.emulateNetworkConditions` via the Chrome DevTools Protocol.

Reviewer: ```suggestion
CPU throttling allows you to simulate how a page performs on slower mobile devices. This can be done using Puppeteer’s `page.emulateNetworkConditions` API.
```
